packages/create-app/templates/default-app/packages/app/package.json.hbs

```diff
@@ -61,6 +61,7 @@
     "@types/node": "^16.11.26",
     "@types/react-dom": "*",
     "cross-env": "^7.0.0",
+    "cross-fetch": "^3.1.6",
```

Comment:
```suggestion
    "cross-fetch": "^3.1.5",
```

i know it doesn't make any practical difference here, but let's keep them all unified!

Reply:
Feel it makes so little difference that I cba updating the PR and want to get the release out 😅